packages/components/src/_styles/input.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Internal dependencies
+ */
+import { reduceMotion } from './reduceMotion';
+import { borderRadius } from './borderRadius';
+
+export const inputPlaceholder = () => `
+	// Use opacity to work in various editor styles.
+	&::-webkit-input-placeholder {
+		color: rgba(#0e1c2e, 0.62);
```

Comment:
Hard coding colors for now rather than re-introducing the same color variables in SASS because I'd like to introduce an alternate theming system in a future PR.

Reply:
👋 Hallooo!! Just a heads up, you'll have to use the `rgba()` function for this color:
https://github.com/WordPress/gutenberg/blob/master/packages/components/src/utils/colors.js#L22

`rgba(HEX, alpha)` only works in Sass 😊

Reply:
Good catch. Thanks!